src/main/java/com/softdevsix/mappers/requests/ProjectCreateRequest.java

```diff
@@ -0,0 +1,4 @@
+package com.softdevsix.mappers.requests;
+
+public class ProjectCreateRequest {
```

Comment:
DTO's shouldn't this be in the application or infrastructure layer? and also be immutable?

Reply:
It just the directory structure what I did and pick between a class or record was not an explicit part of it.